app/jobs/risc_delivery_job.rb

```diff
@@ -52,6 +52,13 @@ def perform(
         }.to_json,
       )
     end
+
+    track_event(
```

Comment:
it seems like each of these `track_event` calls lines up with a `Rails.logger.warn` and a lot of similar values, do you think it would make sense to combine these? maybe we add a `warn_event_name` or something key to capture what's different?

Reply:
i am happy to do that -- i wasn't sure what if anything the logging is used for which made me reticent to touch it. if it's just being used for debugging purposes (that's what i used it for) i'm also curious if we can just add the transport type to the event and axe the `warn` entirely. wdyt?

Reply:
yeah AFAIK that logging isn't used entirely and we tend to look at events.log first anways, so this covers that potential use case anyways, maybe just axe the `Rails.logger.warn` yeah

Reply:
yay i love deleting code

Reply:
removed the warn in https://github.com/18F/identity-idp/pull/9750/commits/11c6a8115be4dc45e3bff9985994a6d744d56d01